### ISSUE TITLE
'ls' display bug

### ISSUE THREAD

**Issue by ZengGengSen** [uutils/coreutils#6440]
When using ssh to connect to a terminal on another windows system, there is a problem with the file displayed

![image](https://github.com/uutils/coreutils/assets/54354608/21958bcd-2868-4920-9f7a-c7a4e1da0899)

**Comment by tertsdiepraam** [comment]
Hmmm, it's a miscalculation around the quotes probably. There were some recent changes around that

**Comment by ZengGengSen** [comment]
```shell
 Apps        Drivers              PerfLogs                Python27     Qt           Users                     del.bat   lua-5.3.5_Win64_bin
 Dell        Intel               'Program Files'         Python38     Qt5          Windows                   extres    temp
 Downloads   KDubaSoftDownloads  'Program Files (x86)'   QMDownload   Strawberry   aow-install-apk-100.xml   global
 ```

**Comment by ZengGengSen** [comment]
In some directories, this is not the case
```shell
forest on  C:/
# ls
 Apps        Drivers              PerfLogs                Python27     Qt           Users                     del.bat   lua-5.3.5_Win64_bin
 Dell        Intel               'Program Files'         Python38     Qt5          Windows                   extres    temp
 Downloads   KDubaSoftDownloads  'Program Files (x86)'   QMDownload   Strawberry   aow-install-apk-100.xml   global

forest on  C:/
# cd /D D:\work\work2405\emulators\libretro\libretro-fceumm\                                                                                                
forest on  D:/work/work2405/emulators/libretro/libretro-fceumm  master
# ls
.github     .gitlab-ci.yml  Authors  Makefile         Makefile.libretro  Tupfile       changelog.txt  intl  mason.build  whatsnew.txt
.gitignore  .travis.yml     Copying  Makefile.common  README.md          appveyor.yml  control        jni   src          zzz_todo.txt

forest on  D:/work/work2405/emulators/libretro/libretro-fceumm  master
#
```

**Comment by ZengGengSen** [comment]
I tried to add a file with a space in it, and this happened

```shell
 forest on  D:/work/work2405/emulators/libretro/libretro-fceumm  master
 # ls
 .github      .gitlab-ci.yml   Authors   Makefile          Makefile.libretro   Tupfile   appveyor.yml    control   jni           src            zzz_todo.txt
 .gitignore   .travis.yml      Copying   Makefile.common   README.md          'a b'     changelog.txt   intl      mason.build   whatsnew.txt
 ```

**Comment by ZengGengSen** [comment]
In some cases, the file name of the previous line is output to the next line
```shell
 forest on  ~
 # ls
 .AndroidStudio3.6   .bash_history   .jdks               .wget-hsts     Downloads   Nox_share       UIDowner                              delete       source
 .BigNox             .config         .lesshst           '3D Objects'   Favorites   OneDrive        Videos                                inittk.ini   useruid.
ini
 .VirtualBox         .designer       .oracle_jre_usage   Contacts       Links       Pictures       'Yinxiang Biji'                       inst.ini     vmlogs
 .WebStorm2019.2     .gitconfig      .ssh                Desktop        Music      'Saved Games'   ansel                                 nuuid.ini    约战竞技
场
 .android            .gradle         .vscode             Documents      NSIS        Searches        d4ac4633ebd6440fa397b84f1bc94a3c.7z   other
```

**Comment by 2205u** [comment]
I'll try looking into this

**Comment by Its-Just-Nans** [comment]
Tried to reproduce but seems like 
- https://github.com/uutils/coreutils/pull/6402 fixed it ?!

**Comment by RenjiSann** [comment]
@ZengGengSen can you share the version of uutils with which you encountered the bug ?
This should have been addressed in 0.0.27, so it is normal to encounter the bug in previous versions.

**Comment by ZengGengSen** [comment]
The bug has been resolved in 0.0.27
